ansible/roles/fmn-dev/files/fmn-worker.service

```diff
@@ -1,11 +0,0 @@
-[Unit]
```

Comment:
Why not keep it called ```fmn-worker.service```? Celery is just an implementation detail.

Reply:
It turns out the celery workers do not like the missing fields from the pre-celery unprocessed messages (no task ID, for example). I guess it doesn't really matter in the development environment, but some sort of careful migration strategy is necessary that involves shutting down old services and starting new services as queues drain. I'm going to write up docs as soon as I figure out a good approach and test it thoroughly.